core/src/test/java/com/klaytn/caver/common/ContractTest.java

```diff
@@ -780,9 +793,51 @@ public void getPastEvent() throws IOException, ClassNotFoundException {
 
         List<KlayLogs.LogResult> logResults = logs.getLogs();
 
+
         KlayLogs.LogObject logObject = (KlayLogs.LogObject)logResults.get(0);
         KlayLogs.Log log = logObject.get();
 
         EventValues eventValues = ABI.decodeLog(contractIOTypes, log.getData(), log.getTopics());
     }
+
+    @Test
+    public void onceTest() throws IOException, TransactionException {
+        WebSocketService web3jService = new WebSocketService("ws://localhost:8552", false);
```

Comment:
```suggestion
        WebSocketService websocketService = new WebSocketService("ws://localhost:8552", false);
```
Can we change this name to `websocketService`?